NMEA2000.h

```diff
@@ -345,6 +349,13 @@ inline void SetN2kProductInformation(tN2kMsg &N2kMsg, unsigned int N2kVersion, u
                   SertificationLevel,LoadEquivalency);
 }
 
+
+void SetN2kPGN59904(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN);
+
+inline void SetN2kPGNISORequest(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN) {
```

Comment:
Good point, Requester is better.

I think you are right about sending NAK only for non broadcasts. But can not say for sure, since I do not have manuals. There would be no sense for responding NAK to broadcast, but not all on N2k has sense anyway.